tesseract_process_managers/include/tesseract_process_managers/core/process_planning_server.h

```diff
@@ -87,6 +90,27 @@ class ProcessPlanningServer
   ProcessPlanningServer& operator=(ProcessPlanningServer&&) = default;
 #endif  // SWIG
 
+  /**
+   * @brief Add a executors (thread pool) under the provided name
+   * @brief This creates a taskflow executor with the provided number of threads
+   * @param name The name of the thread pool
+   * @param n The number of threads
+   */
+  void addExecutor(const std::string& name, size_t n);
```

Comment:
Should we just pass in the executor? Then you could share executors between servers if you wanted.

Reply:
Create idea. I added another method to support this.